fdbserver/BlobManager.actor.cpp

```diff
@@ -338,6 +358,7 @@ ACTOR Future<Void> rangeAssigner(BlobManagerData* bmData) {
 
 			workerId = pickWorkerForAssign(bmData);
 			bmData->workerAssignments.insert(assignment.keyRange, workerId);
+			bmData->workerStats[workerId].numGranulesAssigned += 1;
```

Comment:
Also decrement numGranulesAssigned when iterating over currentAssignments in revoke.